waypoints/Switzerland.cup

```diff
@@ -31,7 +31,7 @@
 "Depot De Sales",,CH,4638.000N,00659.100E,838.0m,3,050,300.0m,,"Landefeld"
 "Disentis O",,CH,4642.183N,00851.433E,1150.0m,3,130,300.0m,,"Landefeld"
 "Disentis W",,CH,4642.000N,00851.150E,1170.0m,3,070,300.0m,,"Landefeld"
-"Dittingen Gld",,CH,4726.350N,00729.400E,540.0m,4,110,620.0m,118.175,"Flugplatz"
+"Dittingen Gld",,CH,4726.350N,00729.400E,540.0m,4,110,620.0m,118.18,"Flugplatz"

```

Comment:
This should probably be `118.180`, right?

Reply:
Now that you ask me, i'm unsure its 118.018 or 118.180.. will clarify.

Reply:
I agree with @Turbo87 that it should be `118.180` (the "official" channel identifier) instead of `118.18`

There is this small oddity about _channels_ and _frequencies_. Your 8.33 kHz radio displays channels instead of frequencies. There is no channel  `.018`. I was not able to quickly find an official document from Eurocontrol but this link to make the principle clear:

http://g3asr.co.uk/calculators/833kHz.htm
> Controller says:   Actual frequency to set on receiver:
> 118.005            118.0000
> 118.010            118.0083
> **118.015            118.0166**
> 118.030            118.0250
> […]
> 118.165            118.1666
> **118.180            118.1750**
> 118.185            118.1833
> 118.190            118.1916

Reply:
So I checked this and it indeed is 180.180